caffe2/operators/fully_connected_op.h

```diff
@@ -208,9 +208,9 @@ class FullyConnectedGradientOp : public Operator<Context> {
     CAFFE_ENFORCE(K * N == W.numel(), dimErrorString());
 
     auto* dW = Output(0);
```

Comment:
I think you missed this one

Reply:
Oh, this one is using ResizeLike, it will be in another codemod